Harvest.Web/Views/Shared/_Layout.cshtml

```diff
@@ -88,7 +88,7 @@
                     @if (await UserService.HasAccess(AccessCodes.WorkerAccess))
                     {
                         <li class="nav-item">
-                            <a asp-controller="Expense" asp-action="Entry" class="nav-link">Expenses</a>
+                            <a href="/expense/entry" class="nav-link">Expenses</a>
```

Comment:
I think this will still cause a refresh. Maybe call history.push in `onClick` handler and cancel default behavior?

Reply:
Yea that's ok, the change wasn't to help with the refresh, it was because the controller route would no longer work. Since the page is served from outside of react I think the full refresh is fine and matches the behavior of all the other links up there.